src/main.py

```diff
@@ -34,27 +44,41 @@ def get_video_name(video_path):
         classes = f.read().rstrip('\n').split('\n')
 except FileNotFoundError as e:
     print(f"Error: Classes file '{classesFile}' not found: {e}")
-    sys.exit(1)
 
-modelConfiguration = "./darknet-yolov3.cfg"
-modelWeights = "./model.weights"
+
+modelConfiguration = "../src/darknet-yolov3.cfg"
+modelWeights = "../src/model.weights"
 
 try:
     net = cv.dnn.readNetFromDarknet(modelConfiguration, modelWeights)
     net.setPreferableBackend(cv.dnn.DNN_BACKEND_OPENCV)
     net.setPreferableTarget(cv.dnn.DNN_TARGET_CPU)
 except cv.error as e:
     print(f"Error loading model: {e}")
-    sys.exit(1)
+   
+
 
 def getOutputsNames(net):
+    """
+    Obtient les noms des couches de sortie du réseau.
+    
+    Args:
+        net: Le réseau de neurones.
+        
+    Returns:
+        List[str]: Liste des noms des couches de sortie.
+    """
     try:
         layerNames = net.getLayerNames()
         return [layerNames[i - 1] for i in net.getUnconnectedOutLayers()]
     except Exception as e:
         print("Erreur lors de la récupération des noms de sortie:", e)
 
+
 class VehicleClassifier(nn.Module):
+    """
+    Classe pour le classificateur de véhicules.
+    """
     def __init__(self, num_classes=2):

```

Comment:
Instead of loading the models in the init here try creating a class per example "model loader" for loading the model and resnet and passing an object to reuse in the whole code, withou reload it for every frame process.
Then, you'll use this ModelLoader instance to access the models for processing frames.